ctapipe/core/tests/test_provenance.py

```diff
@@ -19,6 +19,8 @@ def provenance():
     prov.finish_activity("test1")
     return prov
 
+def test_provenance_activity_names(provenance):
+    prov = provenance
```

Comment:
I guess this line is not needed, could just use 
```py
assert set(provenance.finished_activity_names) == {"test2", "test1"}
```

Reply:
done